core/domain/exp_fetchers_test.py

```diff
@@ -85,23 +98,28 @@ def test_get_exploration_summary_by_id(self):
         self.assertIsNotNone(exp_summary)
         self.assertEqual(exp_summary.id, self.EXP_1_ID)
 
-    def test_get_exploration_summaries_from_models(self):
+    def test_get_exploration_summaries_from_models(self) -> None:
         exp_ids = [self.EXP_1_ID, self.EXP_2_ID, self.EXP_3_ID]
-        exp_summaries = exp_fetchers.get_exploration_summaries_matching_ids(
-            exp_ids
-        )
+        strict_exp_summary_models = []
+        exp_summary_models = exp_models.ExpSummaryModel.get_multi(exp_ids)
+        for model in exp_summary_models:
+            # Ruling out the possibility of None for mypy type checking.
+            assert model is not None
+            strict_exp_summary_models.append(model)
         exp_summary_dict = (
-            exp_fetchers.get_exploration_summaries_from_models(exp_summaries)
+            exp_fetchers.get_exploration_summaries_from_models(
+                strict_exp_summary_models
+            )
```

Comment:
This test case is changed a bit!, because previously we are passing `exp_summaries` (a list of ExplorationSummary domain object ) to `get_exploration_summaries_from_models` ( this method can only accept List of ExpSummaryModel not ExpSummary domain object )